src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1283,6 +1284,20 @@ namespace DataSurfaces {
         }
     };
 
+    struct SurfaceIncidentSolarMultiplier
+    {
+        // Members
+        std::string Name;
+        int SurfaceIdx; // surface index
+        Real64 Scaler;
+        int SchedPtr; // schedule pointer
+
+        // Default Constructor
```

Comment:
You can add defaults to the declaration and delete the default constructor. See how `SurfaceData` is declared.